lib/config.js

```diff
@@ -407,8 +407,14 @@ Config.prototype.getConfig = function(filePath) {
     // Step 8: Merge in command line globals
     config = util.mergeConfigs(config, { globals: this.globals });
 
+    // Step 9: Merge in command line plugins
+    if (this.options.plugins) {
```

Comment:
Sorry, not sure I understand. Won't `config` always exist because of https://github.com/eslint/eslint/pull/3389/files#diff-f7bf2b665273dfa66ffa4ad7c0a52bb9L378?